warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__services_guideline_index.sql

```diff
@@ -11,6 +11,9 @@ int_gtfs_quality__service_guideline_index AS (
         service_key,
     FROM daily_assessment_candidate_entities
     WHERE date < CURRENT_DATE
+    -- Since this is the earliest date in fct_observed_trips, it's the earliest date we currently need
```

Comment:
This kind of makes me wonder if we need to add first checked date to `fct_implemented_checks` to help explain why some checks may not be available on all dates.

Reply:
Yes I think that's a good idea. Right now I kind of only guarantee that checks make sense for the current day looking forward -  consistency going backwards isn't really there.